bridge/bridge/api/static/openapi/openapi.yaml

```
@@ -0,0 +1,919 @@
+      description: |
+        Estimates the result of an unwrap operation, including fees and expected payout.
+        Unwrap operations are only available in wrap and stake modes and are not supported when the
+        bridge is configured for swap mode.
+      tags:
+        - Unwrap
+      operationId: prepareUnwrap
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PrepareRequest'
+      responses:
+        200:
+          description: Estimated unwrap transaction details, including net payout after fees.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PrepareResponse'
+        400:
+          description: Invalid unwrap preparation parameters or constraints violated.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        500:
+          description: Unexpected internal error while estimating unwrap payout.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+components:
+  parameters:
+    Address:
+      name: address
+      in: path
+      required: true
+      description: |
+        Address used for filtering wrap or unwrap requests and errors.
+        The exact format of addresses depends on the configured network (e.g. Ethereum, Symbol, NEM).
+        Clients should validate addresses according to the blockchain in use.
+      schema:
+        type: string
+        example: 0x12ab34cd56ef7890ab12cd34ef567890ab12cd34
+
+    TransactionHash:
+      name: transaction_hash
+      in: path
+      required: true
+      description: Transaction hash used to filter wrap or unwrap requests and errors.
+      schema:
+        type: string
+        pattern: '^[0-9A-Fa-f]{64}$'
+        example: 3f5b8f9a2bb2f6a9c3d2b4f0a1e2c3d4e5f6a7b8c9d0e1f2a3b4c5d6e7f8a90
+
+    Offset:
+      name: offset
+      in: query
+      required: false
+      description: Number of items to skip for pagination.
+      schema:
+        type: integer
+        minimum: 0
+        default: 0
+
+    Limit:
+      name: limit
+      in: query
+      required: false
+      description: Maximum number of items to return.
+      schema:
+        type: integer
+        minimum: 0
+        default: 25
+
+    Sort:
+      name: sort
+      in: query
+      required: false
+      description: |
+        Sort order for results by request transaction height. Values: `1` (ascending) or `0` (descending).
+      schema:
+        type: integer
+        enum:
+          - 1
+          - 0
+        default: 1
+
+    PayoutStatus:
+      name: payout_status
+      in: query
+      required: false
+      description: Filter by payout processing status (0 – unprocessed, 1 - sent, 2 - completed, 3 - failed).
```

```suggestion
      description: "Filter by payout processing status: `0` (unprocessed), `1` (sent), `2` (completed) or `3` (failed)."
```

To be consistent with the previous field.